.github/workflows/gh-ci.yaml

```diff
@@ -171,6 +171,12 @@ jobs:
         build-tests: false
         build-docs: true
 
+    - name: doctests
+      if: github.event_name == 'pull_request'
+      continue-on-error: true
+      run: |
+        cd package && python setup.py build_sphinx -b doctest --keep-going
```

Comment:
I think `python setup.py build_sphinx` has been deprecated because setup.py does not want to be a build system (or something like that). Is there an alternative for invoking doctests??

Reply:
I _think_ setuptools isn't as harsh on the deprecation here, but I'll try switching to sphinx-build instead and see what happens.

Reply:
That should do the trick. I'd probably say we should do the docs building & deployment elsewhere? I'm a bit apprehensive of changing that right now and then unleashing a whole other set of problems.